README.md

```
@@ -63,8 +63,11 @@ You'll also need a slf4j API implementation at runtime, so please add appropriat
 You may find working example of [client][client-example]
 and [server][server-example] applications in module ["examples"][examples-module].
 
-I recommend running server with Concurrent Mark Sweep Collector enabled: `-XX:+UseConcMarkSweepGC`
-and increased Survivor spaces (`-XX:SurvivorRatio=4`).
+### JVM notes
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)